src/Mvc/src/Microsoft.AspNetCore.Mvc.Razor/Compilation/RazorViewAttribute.cs

```diff
@@ -1,28 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-
-namespace Microsoft.AspNetCore.Mvc.Razor.Compilation
-{
-    [Obsolete("This attribute has been superseded by RazorCompiledItem and will not be used by the runtime.")]
-    [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
-    public class RazorViewAttribute : Attribute
```

Comment:
This will cause using anything compiled against 2.0 to fail with a `TypeLoadException`. This was also **just** marked obsolete

Reply:
I think we need to take a lot more care with removing obsolete things that appear in user-code and build outputs of user code.

Reply:
Ah, I took the presence of the release/2.2 link above this specific item verbatim in the issue. Figured it was obsoleted there. Either way I think you have a fair point about the type of complications this would cause to remove an API like this.